### DISCUSSION TITLE
Show all schemas with all Tables

### DISCUSSION THREAD

**Discussion by julenoyonperal** [duckdb/duckdb#14282] [Q&A]
Hi, 

I have created two schemas with 2 tables in each schema in my local duckdb database. See below:
```
sch1.data_jan_24
sch1.data_feb_24
sch2.data_summary_jan_24
sch2.data_summary_feb_24

```
How can i show all the schemas with all the tables? If i just create a table `data_jan_24` and then I use `show tables`, it shows the table, but if I create a schema it doesn't show the schema or the tables.

**Comment by JulenLidl** [comment]
Found the answer: 

```
SELECT table_name 
FROM information_schema.tables 
WHERE table_schema = 'schema_name';
```
